### ISSUE TITLE
Support AWS GovCloud

### ISSUE THREAD

**Issue by bondsbw** [tilfin/aws-extend-switch-roles#75]
Thank you for your work building this extension.  I have several AWS accounts and roles and use it very often.

AWS recently added the standard role-switcher UI to AWS GovCloud.  This is a request to add support for AWS GovCloud regions.

**Comment by tilfin** [comment]
I am not familiar with AWS GovCloud, but what kind of the change should I make for it?

**Comment by bondsbw** [comment]
I was able to get it running locally, though I didn't test it very thoroughly.  My changes involved two files:

- manifest.json
  - added entries for `https://console.amazonaws-us-gov.com/*` and `https://*.console.amazonaws-us-gov.com/*`

- content.js
  - replaced the sign-in URL with `https://signin.amazonaws-us-gov.com/switchrole`

For the sign-in URL I used the current region prefix `us-gov-` to make the determination of which URL to use.  You might consider using the hostname instead.
